### ISSUE TITLE
False positive E1111 assignment-from-no-return for `numpy.arc*`

### ISSUE THREAD

**Issue by camminady** [PyCQA/pylint#2850]
I am getting an E1111 (assignment-from-no-return) when using:
`numpy.arccos`,`numpy.arcsin`,`numpy.arctan`,`numpy.cosh`,`numpy.sinh`,`numpy.tanh`

### Steps to reproduce
Example file `myfunc.py`:

```
"""Example to demonstrate the arccos E1111 situation."""
from numpy import arccos


def myfunc():
    """E1111 for line below"""
    something = arccos(0.4)
    return something
```
Then you just run `pylint myfunc.py`.

### Current behavior

```
pylint myfunc.py
************* Module tools.myfunc
myfunc.py:7:4: E1111: Assigning result of a function call,
where the function has no return (assignment-from-no-return)

--------------------------------------------------------------------
Your code has been rated at -2.50/10 
```

### Expected behavior
```
pylint myfunc.py

---------------------------------------------------------------------
Your code has been rated at 10.00/10 

```
### pylint --version output
```
pylint --version
pylint 2.2.2
astroid 2.1.0
Python 3.7.3 (default, Mar 26 2019, 21:43:19) 
[GCC 8.2.1 20181127]
```
### numpy version
```
python -c "import numpy; print(numpy.version.version)"                                             
1.16.0
```

**Comment by PCManticore** [comment]
Hey @camminady This is fixed with the latest `pylint==2.3.1`.
